lib/logstash/outputs/elasticsearch.rb

```diff
@@ -531,4 +540,15 @@ def check_action_validity
 
     raise LogStash::ConfigurationError, "Action '#{@action}' is invalid! Pick one of #{valid_actions} or use a sprintf style statement"
   end
+
+  def check_dlq_custom_codes
+    intersection = dlq_custom_codes & DOC_DLQ_CODES
+    raise LogStash::ConfigurationError, "dlq_custom_codes contains error codes already used: #{intersection}" unless intersection.empty?
+
+    intersection = dlq_custom_codes & DOC_SUCCESS_CODES
+    raise LogStash::ConfigurationError, "dlq_custom_codes contains error codes already defined as success: #{intersection}" unless intersection.empty?
```

Comment:
suggest a rephase for you to consider "#{intersection} are success codes which cannot be redefined in dlq_custom_codes"

Reply:
Fixed by 5a81f0a328dbdcaa683cf7fd059677f3791a0dbe